system/systemd.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/python
```

Comment:
^^^ 
I'd rather not use the 'systemd' name space yet. (Maybe... 'systemctl' or 'systemctl_unit' or 'systemd_service' or 'systemd_unit')